src/components/chartRepo/chartRepo.service.tsx

```diff
@@ -19,4 +19,9 @@ export const saveChartProviderConfig = (request: any, id: any) => {
 export const validateChartRepoConfiguration = (request: any):Promise<any> => {
     const URL = `${Routes.CHART_AVAILABLE}/${Routes.CHART_REPO}/validate`;
     return post(URL, request);
-  }
\ No newline at end of file
+}
+
+export const reSyncChartRepo = ():Promise<any> => {
+    const URL = `${Routes.CHART_AVAILABLE}/${Routes.CHART_REPO}/${Routes.CHART_RESYNC}`;
+    return get(URL);
```

Comment:
can be post (not fetching any data)

Reply:
this needs to fixed at backend, ll have this and other changes done at the backend, post which will make changes here also

Reply:
This is pending